src/sentry/api/bases/organization.py

```diff
@@ -195,6 +195,23 @@ def get_projects(
             project_ids, request, organization, force_global_perms, include_all_accessible
         )
 
+    def get_projects_by_slugs(self, request, organization, **kwargs):
+        slugs = request.GET.getlist("projectSlug")
+        if slugs == ALL_ACCESS_PROJECTS_SLUG:
+            p_ids = ALL_ACCESS_PROJECTS
+        else:
+            projects = Project.objects.filter(
+                organization=organization, slug__in=slugs
+            ).values_list("id", flat=True)
+            p_ids = set(projects)
+
+        return self.get_projects(
```

Comment:
Note - `get_projects` is doing a lot of project validations as well which is why we're simply passing through to this method.